sample/TKEndorseSample.m

```diff
@@ -177,25 +169,27 @@ - (void)testEndorseTransferToken {
     }];
     
     NSString *accountId = self.payerAccountSync.id;
-    // pisp endore and add key begin snippet to include in docs
+    // pisp create and endorse token begin snippet to include in docs
     [member
      getNotification:notificationId
      onSuccess:^(Notification *notification) {
-         if ([notification.content.type isEqualToString:@"ENDORSE_AND_ADD_KEY"]) {
-             EndorseAndAddKey *content = [TKJson
-                                          deserializeMessageOfClass:[EndorseAndAddKey class]
+         if ([notification.content.type isEqualToString:@"CREATE_AND_ENDORSE_TOKEN"]) {
+             CreateAndEndorseToken *content = [TKJson
+                                          deserializeMessageOfClass:[CreateAndEndorseToken class]
                                           fromJSON:notification.content.payload];
-             NSDecimalNumber *amount = [NSDecimalNumber decimalNumberWithString:content.payload.transfer.lifetimeAmount];
-             TransferTokenBuilder *builder =[member createTransferToken:amount currency:content.payload.transfer.currency];
-             builder.toAlias = content.payload.to.alias;
-             builder.toMemberId = content.payload.to.id_p;
+             NSDecimalNumber *amount = [NSDecimalNumber decimalNumberWithString:content.tokenRequest.requestPayload.transferBody.lifetimeAmount];
+             TransferTokenBuilder *builder =[member createTransferToken:amount currency:content.tokenRequest.requestPayload.transferBody.currency];
+             builder.toMemberId = content.tokenRequest.requestPayload.to.id_p;
+             if (content.tokenRequest.requestPayload.to.hasAlias) {
+                 builder.toAlias = content.tokenRequest.requestPayload.to.alias;
+             }
              builder.accountId = accountId;
-             builder.refId = content.payload.refId;
+             builder.refId = [TKUtil nonce];
```

Comment:
In the old flow, merchants will look up refId so that they can know what this token is for? Is this flow changed? if not we might need this refId instead of nonce.